app/pb_kits/playbook/pb_typeahead/docs/_typeahead_default.html.erb

```diff
@@ -35,7 +36,7 @@
         event.detail.setResults((result.items || []).map((user) => {
           const wrapper = resultOptionTemplate.content.cloneNode(true)
           wrapper.querySelector('slot[name="name"]').replaceWith(user.login)
-          wrapper.querySelector('img').setAttribute("src", user.avatar_url)
+          wrapper.querySelector('img').dataset.src = user.avatar_url
```

Comment:
We use [lazysizes](https://github.com/aFarkas/lazysizes) to load images, which the `user` kit that this kit implements will need. This means you shouldn't be setting `src` but rather `dataset.src`. This allows lazysizes to work correctly.